README.md

```diff
@@ -41,39 +47,170 @@ yarn compile
 
 ### Deploying Contracts
 
-To deploy contracts, we need to run `yarn deploy` command.
+To deploy contracts, we need to run `yarn deploy:mainnet` or `yarn deploy:testnet`
+command which will use the interactive shell to select options if not used in
+the environment variable.
 
 ```shell
 
-# Deploying `cJPY` and `LearnToEarn` in the testnet
+# Deploying contracts in the testnet
+
+yarn deploy:testnet
+
+Please select one of the contracts below:
+
+0. CitNFT
+1. CJPY
+2. Faucet
+3. LearnToEarn
```

Comment:
It seems that `/scripts/deploy.ts` only encounters an issue when deploying the LearnToEarn contract.  

This is because the registry contract needs to grant the LearnToEarn contract address, as shown here:  
https://github.com/henkaku-center/cit-coin-contract/blob/40d33df4a836c30a0c9f44fff51b47839e595fde/scripts/deployLearnToEarn.ts#L18-L19  

While `scripts/deployLearnToEarn.ts` can handle the granting process, `/scripts/deploy.ts` currently cannot, right?  

Therefore, we need to make sure that `/scripts/deploy.ts` can also perform the granting process.